app/assets/javascripts/discourse/app/initializers/inject-objects.js

```diff
@@ -1,9 +1,41 @@
 import { setDefaultOwner } from "discourse-common/lib/get-owner";
+import { isLegacyEmber } from "discourse-common/config/environment";
+import User from "discourse/models/user";
+import deprecated from "discourse-common/lib/deprecated";
 
 export default {
   name: "inject-objects",
   initialize(container, app) {
     // This is required for Ember CLI tests to work
     setDefaultOwner(app.__container__);
+
+    // Backwards compatibility for Discourse.SiteSettings and Discourse.User
+    if (!isLegacyEmber()) {
+      app.yolo = "abc";
```

Comment:
```suggestion
```

I feel like you left this in accidentally :)

Reply:
Haha... yup!